drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -127,9 +127,15 @@ removeCollectionQuestion=Do you want to continue and remove the collection?
 removeCollectionWarningText=This action cannot be undone.
 #
 settings=Settings
+coverageReport=Coverage Report
 uploadWarning=Make sure the headers in the selected CSV file have not been modified. Otherwise, the spreadsheet may not be successfully imported.
 importFailedMessage=A CSV file with an incorrect header layout has been imported. Make sure the imported CSV file has unmodified headers.
 importErrorTitle=Error during import
+executed=Executed
+notCovered=notCovered
```

Comment:
I would prefer to use a single word there. Could we for example use _Missed_, _Ignored_, or _Omitted_ (this one is my favorite)?

Reply:
Done (used Omitted)